### PR TITLE
Pull latest from hasura fork.

### PR DESCRIPTION
**Pull request by nicuveo**
This small change updates the `latest` script to pull from our fork of the repo.
This won't work until we do make a first release.

### CODE REVIEW

**Review by ecthiender — APPROVED** [review]
LGTM

**Comment by nicuveo** [comment]
I don't have write access, @ecthiender I'll let you merge?

**Comment by ecthiender** [comment]
@nicuveo sorry I am looking at the reply now 😆 . But I see you have admin access. Shall I merge this?

**Comment by nicuveo** [comment]
I think those are irrelevant now; I have abandoned the idea of running a custom stylish-haskell, we run the stock one and it's good enough. ^^

**Pull request closed without merging**
